### PR TITLE
SONATA sim config updates

### PR DESCRIPTION
**Pull request by WeinaJi**
* Change the default value of "file_name"
* Clarify the description for "spikes_sort_order"

### CODE REVIEW

File: source/sonata_simulation.rst [hunk 1]
```diff
@@ -409,7 +409,7 @@ Collection of dictionaries with each member describing one data collection durin
    dt                             float      Mandatory    Interval between reporting steps in milliseconds. If assigned value smaller than simulation dt, will be set equal to simulation dt.
    start_time                     float      Mandatory    Time to start reporting in milliseconds.
    end_time                       float      Mandatory    Time to stop reporting in milliseconds.
-   file_name                      text       Optional     Specify file name, otherwise takes the form <repname>_SONATA.h5. If '.h5' extension is not included, it will be added.
+   file_name                      text       Optional     Specify file name, otherwise takes the form <repname>.h5. If '.h5' extension is not included, it will be added.
```
Comment (mgeplf) [inline source/sonata_simulation.rst:412]: while we're at it, we may aswell clarify `repname`, as it's not mentioned anywhere else.

Comment (jorblancoa) [reply source/sonata_simulation.rst]: maybe is worth mentioning that the <report_name>.h5 is coming from the key of the dictionary?

Comment (WeinaJi) [reply source/sonata_simulation.rst]: Sure, I've modified the text again. Does it look better?

Comment (jorblancoa) [reply source/sonata_simulation.rst]: LGTM!

**Review by jorblancoa — APPROVED** [review]

**Review by anilbey — APPROVED** [review]
`file_name` change looks good to me. I've never used the `spikes_sort_order` I cannot say something about it.

**Pull request merged**